app/controllers/comment_controller.php

```diff
@@ -55,4 +62,93 @@ public function write()
         $this->set(get_defined_vars());
         $this->render($page);
     }
+
+    public function edit_comment()
+    {
+        check_user_session();
+        $comment = new Comment();
+        $comment_id = get_comment_id_from_url();
```

Comment:
I don't think this function is needed since Param::get already does this for you